server/jetstream_api.go

```diff
@@ -656,8 +656,8 @@ type JSApiMsgGetRequest struct {
 	UpToSeq uint64 `json:"up_to_seq,omitempty"`
 	// Only return messages up to this time.
 	UpToTime *time.Time `json:"up_to_time,omitempty"`
-	// Return messages as of this time.
-	AsOfTime *time.Time `json:"as_of_time,omitempty"`
+	// Return messages as of this start time.
+	StartTime *time.Time `json:"start_time,omitempty"`
```

Comment:
Move this up to where we have fields for batch.

Reply:
This is the multi section.

Reply:
Done.